### PR TITLE
Updated the "client cache minimum retention period"

### PR DESCRIPTION
**Pull request by kerimhanif**
Per email from Mike Terrill this section needed to be changed.

### Summarize the change in the pull request title

Describe your change, specifically *why* you think it's needed.

Fixes #Issue_Number (if necessary)

### CODE REVIEW

**Comment by PRMerger6** [comment]
@kerimhanif : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: sccm/core/plan-design/changes/whats-new-in-version-1906.md [hunk 1]
```diff
@@ -220,7 +220,7 @@ For more information, see [Support Center OneTrace](/sccm/core/support/support-c
 ### Configure client cache minimum retention period
 
 <!--4485509-->
-You can now specify the minimum time for the Configuration Manager client to keep cached content. This client setting controls how long the client stores content in the cache before deleting it. In the **Client cache settings** group of client settings, configure the following setting: **Minimum duration before cached content can be removed (minutes)**.
+You can now specify the minimum time for the Configuration Manager client to keep cached content. This client setting defines the minimum amount of time Configuration Manager agent should wait before it can remove a content from the cache in case more space is needed. In the **Client cache settings** group of client settings, configure the following setting: **Minimum duration before cached content can be removed (minutes)**.
```
Comment (aczechowski) [inline sccm/core/plan-design/changes/whats-new-in-version-1906.md]: @mestew will need to signoff, but here's my suggested rewording:
```suggestion
You can now specify the minimum time for the Configuration Manager client to keep cached content. When the client needs more space in its cache, this client setting defines the minimum amount of time the client waits before it removes content from the cache. In the **Client cache settings** group of client settings, configure the following setting: **Minimum duration before cached content can be removed (minutes)**.
```

Once finalized, we also need to update the same string in the [Client settings](https://docs.microsoft.com/sccm/core/clients/deploy/about-client-settings#minimum-duration-before-cached-content-can-be-removed-minutes) article. @kerimhanif do you want to change that one as well? If not, I can update once this PR is complete.

Comment (kerimhanif) [reply sccm/core/plan-design/changes/whats-new-in-version-1906.md]: I like your suggestion!

**Comment by PRMerger19** [comment]
@mestew : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by mestew** [comment]
#sign-off

**Pull request merged**